pyproject.toml

```diff
@@ -156,15 +157,18 @@ lint.extend-ignore = [
     "F811", # support typing.overload decorator
 ]
 lint.select = [
-    "B",    # flake8-bugbear - https://docs.astral.sh/ruff/rules/#flake8-bugbear-b
-    "C4",   # flake8-comprehensions - https://beta.ruff.rs/docs/rules/#flake8-comprehensions-c4
-    "E",    # pycodestyle errors - https://beta.ruff.rs/docs/rules/#error-e
-    "F",    # pyflakes rules - https://beta.ruff.rs/docs/rules/#pyflakes-f
-    "I",    # isort - https://docs.astral.sh/ruff/rules/#isort-i
-    "W",    # pycodestyle warnings - https://beta.ruff.rs/docs/rules/#warning-w
-    "UP",   # pyupgrade - https://docs.astral.sh/ruff/rules/#pyupgrade-up
-    "I001", # isort
```

Comment:
I'm not sure why isort was left there by the previous template upgrade, I assume it is now fully replaced by ruff. Happy to put it back if I'm wrong though.